src/main/java/com/example/mate_academy/controller/BookController.java

```diff
@@ -0,0 +1,41 @@
+package com.example.mate_academy.controller;
+
+import com.example.mate_academy.model.Book;
+import com.example.mate_academy.service.BookService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/books")
+public class BookController {
```

Comment:
Your task doesn't include web implementation. Let's follow the task requirements.